Formula/util-linux.rb

```diff
@@ -0,0 +1,47 @@
+class UtilLinux < Formula
+  desc "Collection of Linux utilities"
+  homepage "https://github.com/karelzak/util-linux"
+  url "https://www.kernel.org/pub/linux/utils/util-linux/v2.32/util-linux-2.32.tar.xz"
+  sha256 "6c7397abc764e32e8159c2e96042874a190303e77adceb4ac5bd502a272a4734"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--disable-ipcs",        # does not build on macOS
+                          "--disable-ipcrm",       # does not build on macOS
+                          "--disable-wall",        # already comes with macOS
+                          "--disable-libuuid",     # conflicts with ossp-uuid
+                          "--disable-libsmartcols" # macOS already ships 'column'
+
+    system "make", "install"
+
+    ["cal", # these already come with macOS
```

Comment:
Can we use the array as `%w[cal col colcrt …`  without line breaks every time?
And move the comment above the line, saying `Remove binaries already shipped by macOS`

Reply:
pushed